stable/coredns/templates/deployment.yaml

```diff
@@ -84,7 +84,10 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         ports:
-{{ include "coredns.containerPorts" . | indent 8 }}
+{{ include "coredns.containerPorts" . | indent 8 -}}
+        - containerPort: 9153
```

Comment:
We're exposing it as a service here: https://github.com/helm/charts/blob/master/stable/coredns/templates/service-metrics.yaml Any benefit in doing this that I'm missing?

Reply:
The fact that the service is pointing to a port that is NOT exposed on the pod until this PR.

Reply:
Also trying to get this to match more closely to https://github.com/coredns/deployment/blob/master/kubernetes/coredns.yaml.sed#L136 and https://amazon-eks.s3.us-west-2.amazonaws.com/cloudformation/2020-04-21/dns.yaml where they do expose it on the pod. Admittedly those don't have services corresponding to it, but I find it good practice to list out the exposed ports on the pods for a service to point to.

Reply:
As far as I can see, this field is for informational purposes only, exposing happens through a Service object.
See: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core 
```
Exposing a port here gives the system additional information about the network connections a container uses, but is primarily informational. Not specifying a port here DOES NOT prevent that port from being exposed.
```

I'm not objecting to record it here, I fear this might give wrong information if users configure the port differently in their values file. Can you think of a different way, so we don't have to hardcode the port here?